packages/react-native-client/src/hooks/usePeers.ts

```diff
@@ -85,18 +98,100 @@ function addIsActiveToTracks<
     })),
   }));
 }
+
+function getPeerWithDistinguishedTracks<
+  PeerMetadata extends GenericMetadata = GenericMetadata,
+  ServerMetadata extends GenericMetadata = GenericMetadata,
+>(
+  peer: Peer<PeerMetadata, ServerMetadata>,
+): PeerWithTracks<PeerMetadata, ServerMetadata> {
+  const { tracks: peerTracks } = peer;
+  const distinguishedTracks: DistinguishedTracks = {};
+
+  for (const track of peerTracks) {
+    const trackType = track.metadata?.type;
+    if (!trackType) {
```

Comment:
Would it work if we put this `continue` inside `default` case of the `switch` below instead? I think it might be cleaner that way

Reply:
Good point. I can remove this "continue" altogether, it's not needed.